gtfs_realtime_translators/registry/registry.py

```diff
@@ -14,6 +14,7 @@ class TranslatorRegistry:
     TRANSLATORS = {
         'la-metro': LaMetroGtfsRealtimeTranslator,
         'septa-regional-rail': SeptaRegionalRailTranslator,
+        'septa-trolley-lines': SeptaTrolleyLineTranslator,
```

Comment:
nit: let's give this the key `septa` and name the class `SeptaTranslator` because it will also encompass bus & subway in the near future.